lib/src/widgets/chat/video/ds_video_message_bubble.widget.dart

```diff
@@ -57,6 +58,9 @@ class DSVideoMessageBubble extends StatefulWidget {
   // reply id message
   final DSReplyContent? replyContent;
 
+  // Video title, shoud the user want it to be sent.
```

Comment:
My suggestion of comment would be: `/// Title of the video which will be displayed inside the message bubble`.